.github/workflows/kantra.yaml

```
@@ -0,0 +1,123 @@
+          application_private_key: ${{ secrets.KONVEYOR_BOT_KEY }}
+          permissions: "read:packages"
+
+      - name: Create maven settings file
+        if: ${{ !fromJSON(inputs.skip_maven) }}
+        uses: s4u/maven-settings-action@v4.0.0
+        with:
+          servers: '[{"id": "tackle-testapp-public", "username": "{{ vars.KONVEYOR_BOT_ID }}", "password": "${{ steps.get_workflow_token.outputs.token }}"]'
+          path: "${{ runner.temp }}/settings.xml"
+
+      - name: Create kantra config
+        if: ${{ !fromJSON(inputs.skip_maven) }}
```

actionlint (1.7.10)

1st argument of function call is not assignable. "bool" cannot be assigned to "string". called function type is "fromJSON(string) -> any"

(expression)